AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/GraphQLModelBasedTests.swift

```diff
@@ -343,7 +343,7 @@ class GraphQLModelBasedTests: XCTestCase {
         _ = try await Amplify.API.mutate(request: .create(post))
         _ = try await Amplify.API.mutate(request: .update(post))
         
-        try await AsyncExpectation.waitForExpectations([progressInvoked], timeout: TestCommonConstants.networkTimeout)
+        await waitForExpectations([progressInvoked], timeout: TestCommonConstants.networkTimeout)
```

Comment:
Seeing some minor warnings in this file 
`No 'async' operations occur within 'await' expression`
`mutate(request:listener:)' is deprecated: renamed to 'mutate(request:)`